manifests/k/Krisp/Krisp/2.20.3/Krisp.Krisp.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Krisp.Krisp
+PackageVersion: 2.20.3
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://cdn.krisp.ai/mp/2.20.3/win/Krisp_2.20.3.exe
+  InstallerSha256: D83D49481B20F2532890DADD9F1C7F58C26DBC98F79BBB9D1BE4005BCD520B7A
+  ProductCode: '{F43D5850-9929-4CB8-BEB1-AE6A7E8C6BED}'
```

Comment:
```suggestion
  ProductCode: '{528BA019-D2B2-490E-AB12-6CB140B86FDC}'
```

```
Name          : Krisp
Publisher     : Krisp Technologies, Inc.
Version       : 2.20.3
ProductCode   : {528BA019-D2B2-490E-AB12-6CB140B86FDC}
Registry Path : Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Uninstall\{528BA019-D2B2-490E-AB12-6CB140B86FDC}
Uninstall     : winget uninstall --id "{528BA019-D2B2-490E-AB12-6CB140B86FDC}"
```

Reply:
Thanks, I will apply this suggestion can you tell me how to check the product code instead of using `get-wmiobject Win32_Product | Sort-Object Name | Format-Table IdentifyingNumber, Name -AutoSize`.